client/.gitignore

```diff
@@ -2,7 +2,7 @@
 
 # dependencies
 /node_modules
-
+node_modules/*
```

Comment:
You can just use `node_modules` instead of both of these in the root level `.gitignore`.